### PR TITLE
Fixed BLERSSI  pipeline issue(108)

### PR DESCRIPTION
**Pull request by sravi999**

### CODE REVIEW

File: apps/networking/ble-localization/onprem/pipelines/BLERSSI-Pipeline-Deployment.ipynb [hunk 1]
```diff
@@ -24,17 +24,133 @@
      "output_type": "stream",
```
Comment (swiftdiaries) [inline apps/networking/ble-localization/onprem/pipelines/BLERSSI-Pipeline-Deployment.ipynb:24]: <p>Could you please point to where <code>component_file</code>  is set?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (sravi999) [reply apps/networking/ble-localization/onprem/pipelines/BLERSSI-Pipeline-Deployment.ipynb:24]: Hi @swiftdiaries 

We are loading appropriate training component yaml(CPU or GPU) file. 

image

Comment (swiftdiaries) [reply apps/networking/ble-localization/onprem/pipelines/BLERSSI-Pipeline-Deployment.ipynb:24]: Thank you :) Sorry my bad, this cell didn't show up on the ReviewNB for some reason and I missed it

**Comment by swiftdiaries** [comment]
LGTM

**Pull request merged**